Generals/Code/Main/RTS.RC

```diff
@@ -61,7 +61,7 @@ IDI_ApplicationIcon     ICON    DISCARDABLE     "Generals.ico"
 // Bitmap
 //
 
-IDB_LOAD_SCREEN         BITMAP  DISCARDABLE     "Install_Final.bmp"
+// IDB_LOAD_SCREEN         BITMAP  DISCARDABLE     "Install_Final.bmp"
```

Comment:
How about create a 1x1 white `Install_Final.bmp`? This is what I have done with the tools.